fixtures/oscillatorGainParam.ts

```diff
@@ -0,0 +1,385 @@
+import {WebAudioDebuggerEvent} from '../src/chrome/DebuggerWebAudioDomain';
+import {Audion} from '../src/devtools/Types';
+
+export const OSCILLATOR_GAIN_PARAM_EVENTS: Audion.WebAudioEvent[] = [
```

Comment:
What is this for? It looks like a dataset for testing. Can we have an overview comment for this file at the top?